hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileAppendTest.java

```diff
@@ -183,15 +175,6 @@ public static void assertFailsWith(final ResponseCodeEnum status, final Runnable
         assertEquals(status, ex.getStatus());
     }
 
-    private TransactionBody txnWith(final FileAppendTransactionBody op) {
```

Comment:
Why was this removed?

Reply:
It was not being called anywhere

Reply:
it can be removed Joseph I think it is from the really first version